src/Slider/Slider.js

```diff
@@ -838,58 +862,25 @@ module.exports = kind(
 	/**
 	* @private
 	*/
-	downLeft: function (sender, e) {
-		// checking button explicitly because it may be disabled due to value only
-		if (!this.$.buttonLeft.disabled) {
-			e.configureHoldPulse(this.holdConfig);
-			this.previous();
-		}
-	},
-
-	/**
-	* @private
-	*/
-	downRight: function (sender, e) {
-		// checking button explicitly because it may be disabled due to value only
-		if (!this.$.buttonRight.disabled) {
-			e.configureHoldPulse(this.holdConfig);
-			this.next();
-		}
-	},
-
-	/**
-	* @private
-	*/
-	holdLeft: function (sender, event) {
-		if (!this.$.buttonLeft.disabled) {
-			this.previous();
-		}
-	},
-
-	/**
-	* @private
-	*/
-	holdRight: function (sender, event) {
-		if (!this.$.buttonRight.disabled) {
-			this.next();
-		}
-	},
-
-	/**
-	* @private
-	*/
-	configureSpotlightHoldPulse: function (sender, e) {
-		if (e.keyCode === 13) {
-			e.configureHoldPulse(this.holdConfig);
+	jumpButtonTriggered: function (sender, ev) {
+		var isValidEvent = false;
+		if (!sender.disabled) {
+			if ((ev.type == 'down' || ev.type == 'holdpulse') || (ev.keyCode == 13 && ev.type == 'onSpotlightKeyDown')) {
```

Comment:
Hi @DavidUM, no problem! I think that we do not actually need to check for `down` at all - we are only concerned about the `onSpotlightKeyDown` case where the key code is *not* 13. Let me know if there is a case that I am not thinking of. Thanks!